File: ckgame-ts/src/core/calendar/Season.ts
export enum Season {
  SPRING = 'SPRING',
  SUMMER = 'SUMMER',
  AUTUMN = 'AUTUMN',
  WINTER = 'WINTER',
}

export function seasonZh(s: Season): string {
  switch (s) {
    case Season.SPRING: return '春';
    case Season.SUMMER: return '夏';
    case Season.AUTUMN: return '秋';
    case Season.WINTER: return '冬';
  }
}

File: ckgame-rust/src/core/calendar/season.rs
#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
pub enum Season {
    Spring,
    Summer,
    Autumn,
    Winter,
}

pub fn season_name(s: Season) -> &'static str {
    match s {
        Season::Spring => "春",
        Season::Summer => "夏",
        Season::Autumn => "秋",
        Season::Winter => "冬",
    }
}